Item redesign

mvorsteg opened the pull request.

mvorsteg committed:
Creating new items and adding effects to old items

- adding hit sounds for melon
- adding propelon (heat seeking melon)
- adding blue melon (not inspired by anything else ever)
- adding cool effect when using spiedie sauce
- adding cool damage effect when you die

mvorsteg committed:
Adding Jumpsplosion and Back Sentry items

mvorsteg committed:
Enabling speed lines for all racers

mvorsteg committed:
Giving back sentry to all players

adding item icons too

mvorsteg committed:
Update .gitignore

mvorsteg committed:
Backpack mount update, gitignore

mvorsteg committed:
ignoring user settings

mvorsteg committed:
Consolidating player/npc/preview models

Also adding some sound effects to melon

mvorsteg committed:
Enhancements to Back Sentry Item

Now aims better and has a cool UI targeting effect

mvorsteg committed:
Redesign of Vorsteg shirt

grabermtw committed:
Updated Graber & DOUG shirts

grabermtw committed:
Added Speedy Sauce image

grabermtw committed:
Merge branch 'item-redesign' into shirt-redesign

mvorsteg committed:
Redesign of Course 1 billboard

mvorsteg committed:
Creating credits on main menu

mvorsteg committed:
Merge pull request #9 from mvorsteg/shirt-redesign

Redesigned some shirts and added Speedy Sauce image

grabermtw reviewed (changes requested):
Tested it again with Unity 6000.0.30f1, still seeing the same sliding issues with the NPCs unfortunately.

grabermtw committed:
Updated speedy sauce sprite and vorsteg shirt material texture

grabermtw committed:
Fixed a warning about being unable to set linearVelocity on a kinematic rigidbody

grabermtw committed:
Player/NPC audio sources now use 3D sounds

grabermtw committed:
Fixed floating platform bugs

grabermtw committed:
Fixed laser cannon race condition in Course 2

grabermtw committed:
Fixed mystery box height over floating platform

grabermtw committed:
Made the mini laser bolt actually mini

grabermtw committed:
Fixed backpack alignment for each character

grabermtw committed:
Fixed back sentry reticle being visible 180 degrees from target

grabermtw committed:
Fixed collision layers of some backpack items

mvorsteg committed:
Fixing bug where NPCs could not run up a hill

Reverted rigidbodies from Continuous Dynamic to Discrete for all NPCs

mvorsteg committed:
Adding jumplsplosion effects for all movement modes

mvorsteg committed:
Update .gitignore

mvorsteg committed:
Created 3d model for speedy sauce

mvorsteg committed:
Adding stop sign item

Also replacing banana peel with traffic cone

mvorsteg committed:
Adding icons and sounds for new items

mvorsteg committed:
Updating NPC prefabs for item drop points

Also adding new items to mysterybox prefab

mvorsteg committed:
stop sign rotation fix & limiting dynamic stretch distance

mvorsteg committed:
Initial commit for loot table system

Still needs cleanup but should work

mvorsteg committed:
Fixing game breaking issues

-Applying loot table to prefab item boxes, which would not give items without this reference
-Adding item reference to speedy sauce item registry, which is required for the item to be used

grabermtw committed:
Fixed Bestudo backpack items, stop sign 3D sound, tweaked item curves

mvorsteg committed:
Upgrading to Unity 6000.3.2f1

mvorsteg committed:
Adding Polypedia tab to main menu

mvorsteg committed:
Adding item distribution option to race settings

mvorsteg committed:
Fixing bug where quality spinner did not display correct value

mvorsteg committed:
Adding more images

-Background for grid selection entries
-Some detail slides for Dr Desmesmert and Vorsteg
-Any Polypedia details with 2+ image slides will transition between them

mvorsteg committed:
Adding border to capturable area in greenscreen

Should help with cropping

mvorsteg committed:
Fix bug where Buck did not have correct model reference

grabermtw reviewed (approved):
Fantastic
